$OpenBSD: patch-configure,v 1.1 2010/08/30 13:15:49 landry Exp $
Don't force the use of -O2, the ports infrastructure
takes care of that
--- configure.orig	Sun Aug 29 21:50:39 2010
+++ configure	Sun Aug 29 21:52:36 2010
@@ -19530,8 +19530,8 @@ fi
 
 # Add the appropriate 'O' level for optimized builds
 if test "$enable_optimizations" = "yes"; then
-   CXXFLAGS="$CXXFLAGS -O2"
-   CFLAGS="$CFLAGS -O2"
+   CXXFLAGS="$CXXFLAGS"
+   CFLAGS="$CFLAGS"
 else
    CXXFLAGS="$CXXFLAGS -O0"
    CFLAGS="$CFLAGS -O0"
